frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -67,9 +91,7 @@ function NewRunParametersV2(props: NewRunParametersProps) {
     Object.keys(props.specParameters).map(key => {
       if (props.specParameters[key].defaultValue) {
         // TODO(zijianjoy): Make sure to consider all types of parameters.
-        runtimeParametersWithDefault[key] = {
-          string_value: props.specParameters[key].defaultValue,
-        };
+        runtimeParametersWithDefault[key] = props.specParameters[key].defaultValue;
```

Comment:
Does the `defaultValue` match what we expect from the type `specParameters`? Do we need to validate it?

Reply:
https://screenshot.googleplex.com/7MJsG5cg49YTtQe

I think type check is done for the default value before generating YAML file, so it supposed to match the type from `specParameters`.